spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AnnotatedServiceRegistrationBean.java

```diff
@@ -39,7 +43,8 @@
  * >             .setDecorators(LoggingService.newDecorator())
  * >             .setExceptionHandlers(new MyExceptionHandler())
  * >             .setRequestConverters(new MyRequestConverter())
- * >             .setResponseConverters(new MyResponseConverter());
+ * >             .setResponseConverters(new MyResponseConverter())
+ * >             .setExampleRequests(Lists.of(AnnotatedExampleRequest.of("post", "{\"foo\":\"bar\"}")));
```

Comment:
nit: How about using `myMethod` or `hello` instead of `post` for the method name?
Because the method is not an HTTP method but the method in a class, I think `post` can give confusion to a user.